src/js/main.js

```diff
@@ -1,31 +1,69 @@
 (function() {
+  //controller for the item-tabs area of the Listing/Product page
+  $("nav#item-tabs a").click(function(){
+    $(this).addClass("active").siblings().removeClass("active");
+    $("section.tabs").removeClass("active");
+    $($(this).attr("href")).addClass("active");
+  });
 
-//controller for the item-tabs area of the Listing/Product page
-$("nav#item-tabs a").click(function(){
-  $(this).addClass("active").siblings().removeClass("active");
-  $("section.tabs").removeClass("active");
-  $($(this).attr("href")).addClass("active");
-});
 
+  // When you click on the collection button, menu drops down
+  $('.collect-button').on('click', function(){
+    $('.collect-drop-down').toggleClass('show');
+    $('.add-to-list').removeClass('active');
+  });
 
-// When you click on the collection button, menu drops down
-$('.collect-button').on('click', function(){
-  $('.collect-drop-down').toggleClass('show');
-  $('.add-to-list').removeClass('active');
-});
+  // When you click on the favorite button, icon turns red
+  $('.favorite-button').on('click', function(){
+    $(this).toggleClass('user-favorite');
+  });
 
-// When you click on the favorite button, icon turns red
-$('.favorite-button').on('click', function(){
-  $(this).toggleClass('user-favorite');
-});
+  // When you click on the drop down input field, add button appears
+  $('input.list-text').on('click', function(){
+    $('.add-to-list').addClass('active');
+  });
+})(); //end IIFE
 
-// When you click on the drop down input field, add button appears
-$('input.list-text').on('click', function(){
-  $('.add-to-list').addClass('active');
-});
+(function(){ // IIFE for Angular JS!
+  angular.module('TIY-Catalog', [ 'ngRoute' ], function($routeProvider){
+    $routeProvider
+      .when('/trending', { // route
+        // controller: 'name of a controller' || function(){ },
+        // controllerAs: 'identifier name for controller?',
+        // template: 'string HTML template' || function(){
+        //   return 'an HTML template';
+        // },
+        // templateUrl: 'string path' || function(){
+        //   return 'string path';
+        // },
+        templateUrl: 'trending.html'
+        // resolve: see documentation?
+        // redirectTo: 'string path' || function(){
+        //   return 'string path'
+        // }
+      }) // END when(path, route)
+      .when('/listing/0123456789', {
+        templateUrl: 'product.html'
+      })
```

Comment:
I bet we'll need a `controller` here, too... And those `$routeParams`... If we changed `/listing/0123456789` to `/listing/:listingID` guess what shows up in `$routeParams`...!